openlibrary/i18n/messages.pot

```diff
@@ -2556,6 +2556,18 @@ msgstr ""
 msgid "Suggesting Edits"
 msgstr ""
 
+#: nav_foot.html:51
+msgid "Change Website Language"
+msgstr ""
+
+#: nav_foot.html:53
+msgid "Czech"
+msgstr ""
+
+#: nav_foot.html:58
+msgid "Telugu"
+msgstr ""
+
```

Comment:
As I mentioned in another comment, these can probably be skipped (at least for use in the footer).

Reply:
I thought that adding new text in the `messages.pot` file was standard procedure for all new [user visible strings](https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide#internationalization-i18n---for-translators). Should I go ahead and remove them?

Reply:
Yes, please remove them. They don't need to be translated, per my comment above.